crates/txtx-addon-kit/src/types/types.rs

```diff
@@ -291,7 +291,6 @@ impl Value {
         }
     }
 
-    #[deprecated(note = "use `expect_buffer_bytes_result` instead")]
```

Comment:
😡

Reply:
the method `expect_buffer_bytes_result` is missing its point. In rust, a name `expect_` means that you'll be emitting an exception.